src/scalar/mod.rs

```diff
@@ -0,0 +1 @@
+//! This module contains all minimizers for functions with a single argument.
```

Comment:
"single argument" may be a bit misleading, don't you think? How about something along the lines of "Minimizers that minimize functions along one dimension"? This particular sentence also does not "flow", but the main idea is that the number of arguments is not really constrained, it's just that the minimizer will only tweak one "knob" to find the minimum.

Reply:
Agreed, the current formulation is a bit vague. I'll push some changes!